src/libs/ReportUtils.ts

```diff
@@ -2371,7 +2371,7 @@ function getIcons(
 
         // For one transaction IOUs, display a simplified report icon
         if (isOneTransactionReport(report?.reportID ?? '-1')) {
-            return [ownerIcon];
+            return [managerIcon];
```

Comment:
Just to confirm - this change fixes the header avatar of `Report expense` page right?

<img width="1440" alt="Screenshot 2024-10-15 at 8 34 22 PM" src="https://github.com/user-attachments/assets/f9c40f25-9a13-45a7-b931-77c1594e82e3">

Reply:
Yep

Reply:
This should match the IOU case. For IOUs we use managerIcon, for expense reports, ownerIcon